src/bin/glarea/glium_backend.rs

```diff
@@ -0,0 +1,170 @@
+//! # glium_backend utility module
+//!
+//! This module provides some general utility typoes that allow glium to manage
```

Comment:
"This module provides some general utility typoes that allow glium to manage" -> "This module provides some general utility types which allow glium to manage"

Reply:
Appropriate typo, fixed